src/magnifik.js

```diff
@@ -211,10 +216,25 @@ Mobify.UI.Magnifik = (function() {
           , bigHeight = thumbWidth * imgAspect
           , thus = this;
 
-        thus.$canvas.prop('scrollLeft', Math.max(0, Math.min(bigWidth - smallWidth,
-            bigWidth * leftRatio - smallWidth / 2)));
-        thus.$canvas.prop('scrollTop', Math.max(0, Math.min(bigHeight - smallHeight,
-            bigHeight * topRatio - smallHeight / 2)));
+        // Set the desired "scrollLeft" position
+        if (this.options.canvasScrollPosition.horizontal === 'left') {
```

Comment:
we might want to cache these properties to `var horizontal = this.options.canvasScrollPosition.horizontal`, and `this.options.canvasScrollPosition.vertical`